examples/client_server/src/server.rs

```diff
@@ -0,0 +1,40 @@
+use gdnative::prelude::*;
+use gdnative::api::NetworkedMultiplayerENet;
+
+const PORT: i64 = 9876;
+const MAX_CLIENTS: i64 = 1;
+const IN_BANDWIDTH: i64 = 1000;
+const OUT_BANDWIDTH: i64 = 1000;
+
+#[derive(NativeClass)]
+#[inherit(Node)]
+pub struct Server;
+
+#[methods]
+impl Server {
+    fn new(_owner: &Node) -> Self {
+        Self
+    }
+
+    #[export]
+    fn _ready(&mut self, owner: &Node) {
+        let peer = NetworkedMultiplayerENet::new();
+        peer.create_server(
+            PORT,
+            MAX_CLIENTS,
+            IN_BANDWIDTH,
+            OUT_BANDWIDTH
+        ).unwrap();
+
+        if let Some(tree) = owner.get_tree() {
+            let tree = unsafe { tree.assume_safe() };
+            tree.set_network_peer(peer);
+        };
+    }
+
+    #[export(rpc = "master")]
+    fn greet_server(&mut self, owner: &Node, msg: GodotString) {
+        godot_print!("Client says: {}", msg);
+        owner.rpc(GodotString::from_str("return_greeting"), &[Variant::from_str("hello")]);

```

Comment:
I don't think `GodotString::from_str` is necessary here? You should be able to just pass the `&str` to the method.